sktime/transformations/series/detrend/mstl.py

```diff
@@ -0,0 +1,218 @@
+# copyright: sktime developers, BSD-3-Clause License (see LICENSE file)
+"""Implements MSTL."""
+
+__all__ = ["MSTL"]
+__authors__ = ["luca-miniati"]
+
+from typing import Dict, Optional, Sequence, Union
+
+import pandas as pd
+
+from sktime.transformations.base import BaseTransformer
+
+
+class MSTL(BaseTransformer):
+    """Season-Trend decomposition using LOESS for multiple seasonalities.
+
+    Direct interface for `statsmodels.tsa.seasonal.MSTL`.
+
```

Comment:
I think it would be crucial to tell the user what it does in `transform`, i.e., what components it returns, with what column names, etc.